src/openfermion/transforms/__init__.py

```diff
@@ -16,6 +16,7 @@
 from ._bravyi_kitaev import bravyi_kitaev
 from ._binary_code_transform import (binary_code_transform,
                                      dissolve)
+from ._code_transform_functions import *
```

Comment:
please write explicitly what you are importing here so that it is clear to everyone what exists in the namespace of the transforms module.